packages/core/src/wallet.test.ts

```diff
@@ -31,6 +32,15 @@ describe('Wallet', () => {
     expect(result[1]['@context']).toEqual('test');
   });
 
+  it('expect to create default dids', async () => {
+    const dids = await createDIDProvider({wallet}).getAll();
+    expect(dids.length).toBe(1);
+    await wallet.setNetwork('testnet');
+    await new Promise(resolve => setTimeout(resolve, 1000));
+    const dids2 = await createDIDProvider({wallet}).getAll();
+    expect(dids2.length).toBe(1);
```

Comment:
can we assert that `dids[0] != dids2[0]`?

also minor thing, but could we call those variables `mainnetDIDs` and `testnetDIDs` to be explicit?